.env.example

```diff
@@ -1,3 +1,5 @@
 PORT=
 
-MONGO_CONNECTION_STRING=
\ No newline at end of file
+MONGO_CONNECTION_STRING=
+
+SECRET_KEY=Autumn-fairytale
```

Comment:
do not store actual keys here, say `<string>` instead to clarify the format and communicate the actual key value to the team in chat

Reply:
Sure, I got it

Reply:
Fixed 
https://github.com/Autumn-fairytale/AFT-back/pull/34/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR6